pkg/webui/console/containers/application-title-section/status/status.styl

```
@@ -0,0 +1,16 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.status
+  margin-right: $cs.l
```

```suggestion
  nudge('down')
  margin-right: $cs.l
```

If you look closely, the last seen element is not on the same line as the entity counts. Nudging it down is kind of a workaround. Even better would be to adjust the layout so that they will be on the same baseline automatically.